src/main/java/HW_Lesson11/Catalog/Catalog.java

```diff
@@ -0,0 +1,56 @@
+package HW_Lesson11.Catalog;
+
+//Создать класс Catalog с внутренним классом, с помощью объектов которого можно хранить информацию
+//об истории выдачи книги читателям.
+
+public class Catalog {
+    private String visitorName;
+
+    private int days;
+
+    public Catalog(String visitorName, int days){
+        this.visitorName = visitorName;
+//        this.bookNumber = bookNumber;
+        this.days = days;
+    }
+
+    public void returnCatalogInfo() {
+
+        System.out.println(visitorName +" ," +" ,"+days+".");
+    }
+
+    public class BlackList{
```

Comment:
I thought about storing inside this class some information, but not only using for it as util